packages/components/src/stories/SquigglePlayground.stories.tsx

```diff
@@ -248,22 +233,6 @@ export varScatter = Plot.scatter({
   },
 };
 
-export const ImportsWithTooltips: Story = {
```

Comment:
Moved to `Imports.stories.tsx`. TODO: we should move out and group other stories too, because loading the root "SquigglePlayground/Docs" page is getting kinda slow.